apiary-metastore-events/kafka-metastore-events/kafka-events-common/src/main/java/com/expediagroup/apiary/extensions/events/metastore/common/listener/ListenerEventUtils.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.events.metastore.common.listener;
+
+import org.apache.hadoop.hive.common.StatsSetupConst;
+import org.apache.hadoop.hive.metastore.events.ListenerEvent;
+
+public final class ListenerEventUtils {
+
+  private ListenerEventUtils() {}
+
+  /**
+   * This code is borrowed from {@link org.apache.hadoop.hive.metastore.HiveAlterHandler}.
```

Comment:
If so then the copyright notice above needs to be amended to include the original copyright owner and this file added to an exclusion list for the Mycila plugin.

Reply:
So it doesn't look like this class is ever even used anywhere, I have this on my list as a candidate for deletion anyway so could just remove now. @abhimanyugupta07 do you know if this can just be removed altogether?